web/templates/settings/settings_checkbox.hbs

```diff
@@ -1,5 +1,5 @@
 {{#unless (eq render_only false)}}
-<div class="input-group {{#if is_disabled}}control-label-disabled{{/if}}">
+<div class="input-group {{#if is_disabled}}control-label-disabled{{/if}}" id="{{setting_container_id}}">
```

Comment:
We still need the container ID for styling. 

![image](https://user-images.githubusercontent.com/87542880/224990501-254a0218-911c-4157-b79f-acb0407acb38.png)


We can add a wrapper if we do not want to change the `settings_checkbox.hbs` template.

````
<div id="default-stream">
    {{> ../settings/settings_checkbox
      prefix="id_"
      setting_name="is_default_stream"
      is_checked=check_default_stream
      label="Default stream for new users"
      help_link="/help/set-default-streams-for-new-users"
      }}
</div>
````

 I have updated the code for the same.

Reply:
Ok, we can keep this as it is for now. I am not sure about whether we want 25px margin here or not, so can be updated later if required.

Ok, I posted one comment about using class instead of ID for this wrapper element.